test/code-compass-test.el

```diff
@@ -27,9 +27,9 @@
 
 (ert-deftest c/expand-file-name_expanded-file-name ()
   (let ((default-directory "/tmp")
-        (c/path-to-code-compass "somePath"))
```

Comment:
I guess i could of left these as is to make sure the aliases work /shrug
i can revert this commit if desired

Reply:
that is fine as is, according to the melpa linter package-lint we couldn't neither have those aliases. I appreciate a lot the effort to avoid breaking changes, but probably I will have to break for Melpa.